File: C#/258. Add Digits.cs
public class Solution {
    public int AddDigits(int num) {
        while(num/10!=0){
            int sum = 0;
            while(num!=0){
                sum += num % 10;
                num /= 10;
            }
            num = sum;
        }
        return num;
    }
}

File: JavaScript/258. Add Digits.js
/**
 * @param {number} num
 * @return {number}
 */
var addDigits = function(num) {
    while(num>9){
        var sum = 0;
        while(num>0){
            sum += num % 10;
            num = Math.floor(num/10);
        }
        num = sum;
    }
    return num;
};
